ui/page/dexclient/add_dex_modal.go

```diff
@@ -21,34 +18,59 @@ import (
 
 const addDexModalID = "add_dex_modal"
 
-const testDexHost = "dex-test.ssgen.io:7232"
-
 type addDexModal struct {
 	*load.Load
 	modal            *decredmaterial.Modal
-	addDexServer     decredmaterial.Button
+	addDexServerBtn  decredmaterial.Button
 	dexServerAddress decredmaterial.Editor
 	isSending        bool
-	cert             decredmaterial.Editor
-	cancel           decredmaterial.Button
+	cancelBtn        decredmaterial.Button
 	materialLoader   material.LoaderStyle
+	cert             decredmaterial.Editor
+
+	// defaultAppPass is set after login or initialize to continue process of adding a DEX
+	// without requesting password input on UI
+	defaultAppPass string
+	appPassword    decredmaterial.Editor
+	onDexAdded     func(*core.Exchange)
+
+	listServerClickable map[string]*decredmaterial.Clickable
+	selectedServer      string
+	listServerBtn       decredmaterial.Button
+	customServerBtn     decredmaterial.Button
+	isUseCustomServer   bool
```

Comment:
These seem to be related to registering a DEX. Does this PR cover that feature as well?